site/source/docs/getting_started/downloads.rst

```diff
@@ -50,7 +50,12 @@ GitHub and set them as :term:`active <Active Tool/SDK>`:
     ./emsdk activate latest
 
     # Activate PATH and other environment variables in the current terminal
-    source ./emsdk_env.sh
+    source /path/to/emsdk/emsdk_env.sh
+
+    # Verify PATH. This command should return the full path to your emsdk directory
+    which emcc
+
+  .. note:: If you're struggling to get the ``emcc`` command working, add ``export PATH="/path/to/emsdk/upstream/emscripten:$PATH"`` to your ``/home/username/.bashrc`` file.
```

Comment:
`emsdk activate` prints this information, and there is more than one path than you would need to add.

How about: "If you prefer you can update your PATH environment variable yourself.  Th `emsdk activate` command above will print the pathnames you should add".

Reply:
Sounds reasonable. Thanks.